Tools/build/compute-changes.py

```diff
@@ -59,9 +59,7 @@ def compute_changes() -> None:
     target_branch, head_branch = git_branches()
```

Comment:
Maybe rename this to target_ref (and the same for the function and a bunch of other variables)

Reply:
Renamed everything except *target_branch*, as that is still a branch proper